cloud-native-runtimes/install-cnrt.hbs.md

```diff
@@ -65,41 +65,37 @@ To install Cloud Native Runtimes:
 
         ```console
         ---
-        # if deploying on a local cluster such as Kind. Otherwise, you can remove this field
-        provider: local
+        domain_name: example.com
+        ingress:
+        external:
+            namespace: tanzu-system-ingress
+        internal:
+            namespace: tanzu-system-ingress
         ```
 
         >**Note:** For most installations, you can leave the `cnr-values.yaml` empty, and use the default values.
 
         If you are running on a single-node cluster, such as kind or minikube, set the `provider: local`
-        option. This option reduces resource requirements by using a HostPort service instead of a
-        LoadBalancer and reduces the number of replicas.
+        option. This option reduces resources requests for CNR deployments.
+        If you are running on a multinode cluster, do not set `provider`.
 
         Cloud Native Runtimes reuses the existing `tanzu-system-ingress` Contour installation for
```

Comment:
I think we are missing the word `namespace` somewhere in this line. It could be something like `CNRs uses the Contour installation provided by TAP in the `tanzu-system-ingress` for internal and external traffic` or something like that.